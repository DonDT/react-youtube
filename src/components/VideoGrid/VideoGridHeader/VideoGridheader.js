import React from "react";
import "./VideoGridHeader.scss";

export default function VideoGridheader(props) {
  return (
    <div className="video-grid-header">
      <span className="title">{props.title}</span>
    </div>
  );
}
