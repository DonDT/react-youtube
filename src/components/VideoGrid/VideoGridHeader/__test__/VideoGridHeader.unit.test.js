import React from "react";
import { shallow } from "enzyme";
import VideoGridHeader from "../VideoGridheader";

describe("VideoGridHeader", () => {
  test("renders without props", () => {
    const wrapper = shallow(<VideoGridHeader />);
    expect(wrapper).toMatchSnapshot();
  });
  test("test with empty string header", () => {
    const wrapper = shallow(<VideoGridHeader title="" />);
    expect(wrapper).toMatchSnapshot();
  });
  test("renders with title ", () => {
    const wrapper = shallow(<VideoGridHeader title="Trending" />);
    expect(wrapper).toMatchSnapshot();
  });
});
