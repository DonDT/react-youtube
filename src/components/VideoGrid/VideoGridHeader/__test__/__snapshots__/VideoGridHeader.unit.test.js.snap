// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoGridHeader renders with title  1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  >
    Trending
  </span>
</div>
`;

exports[`VideoGridHeader renders without props 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;

exports[`VideoGridHeader test with empty string header 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;
